lib/DropdownMenu/styles.scss

```diff
@@ -120,3 +160,19 @@ $dropdown-menu-font-color: $color-text-base !default;
     outline: none;
   }
 }
+
+.dropdown__additional {
+  display: flex;
+  .item-name {
+    display: block;
+    width: 65%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .item-additional {
```

Comment:
It might be worth sticking to the BEM naming convention here. Unless there's a reason not to?